src/Sushi.Mediakiwi.API/Controllers/Authentication.cs

```diff
@@ -0,0 +1,163 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+using Sushi.Mediakiwi.API.Services;
+using Sushi.Mediakiwi.API.Transport.Requests;
+using Sushi.Mediakiwi.API.Transport.Responses;
+using System;
+
+namespace Sushi.Mediakiwi.API.Controllers
+{
+    [ApiController]
+    [MediakiwiApiAuthorize]
+    [Route(Common.MK_CONTROLLERS_PREFIX + "authentication")]
+    [ResponseCache(NoStore = true, Location = ResponseCacheLocation.None)]
+    public class Authentication : ControllerBase
+    {
+        private readonly IUserService userService;
```

Comment:
rename to _userService, convention for private fields

Reply:
Agreed, adjusted